library_generation/test/generate_library_unit_tests.sh

```diff
@@ -134,6 +134,18 @@ download_protoc_failed_with_invalid_arch_test() {
   assertEquals 1 $((res))
 }
 
+download_protoc_succeed_with_baked_protoc() {
```

Comment:
We need to declare this method at the end of the file in order this method to run.

Reply:
I added it and fixed the corresponding unit test